src/bot.py

```diff
@@ -1,21 +1,22 @@
-import discord
+from discord.ext import commands
+from multiprocessing import Process, Queue
+import apiserver
+import json
 
-client = discord.Client()
-TOKEN = open('src/token.txt', 'r').read()
+bot = commands.Bot(command_prefix='!')
+bot.CONF = json.load(open('src/bot_conf.json', 'r'))
```

Comment:
```suggestion
with open('src/bot_conf.json', 'r') as conf_file:
    bot.CONF = json.load(conf_file) 
```

Asi el fichero se `bot_conf.json` se cierra tras ejecutar la asignacion.